src/python/pants/backend/python/util_rules/pex_from_targets_test.py

```diff
@@ -69,6 +69,7 @@ def rule_runner() -> RuleRunner:
     )
 
 
+@pytest.mark.skip(reason="TODO(#15824)")
```

Comment:
What is the correct way to skip tests?

I got this:
```
Forbidden skipped test - Skipped: TODO(#15824)
```

Reply:
Ah, there it is. That explains why I didn't find anything on `--noskip` in the pytest docs 😆

Reply:
I love how customizable pytest is.